docs/use-cases/greenfield-native.md

```diff
@@ -0,0 +1,113 @@
+---
+id: greenfield-native-app
+title: Native app authentication
+sidebar_label: Native app authentication
+---
+
+## Use case: ACME Corporation is new to security and wants to implement a straightforward authentication solution for their native app
+
+ACME Corporation is building a new subscription-based service from scratch. In the first phase of the project, they implemented a
+browser-based app that integrates with the authentication flows provided by Ory Network. Now they want to extend their coverage,
+by implementing a native app that runs on the Android and iOS mobile platforms. For the initial release of the mobile app, they
+want to include support for social sign-in, as their experience with the browser-based app suggests this will significantly boost
+the rate of new user registrations.
+
+## How Ory Network provides authentication for native apps
+
+The first step ACME Corporation took was to learn about the architecture of Ory Network and how it could provide authentication
+flows for their mobile native app. The following diagram illustrates how a native app integrates with the Ory Network's
+self-service authentication flows and shows some of the main features of the authentication architecture:
+
+![Architecture for native app integrated with Ory Identities](./_static/greenfield-native.svg)
+
+### Self-service flows
+
+To get started with the development of the mobile app and implement a proof of concept quickly, the main thing developers need to
+know about at this point are the self-service flows.
+
+Self-service flows refer to the flows that users perform themselves, providing the essence of the end user's experience. Ory
+Identities implements the following self-service flows for native apps:
+
+- [User Login](https://www.ory.sh/docs/kratos/self-service/flows/user-login)
+- [User Registration](https://www.ory.sh/docs/kratos/self-service/flows/user-registration)
+- [Profile Management](https://www.ory.sh/docs/kratos/self-service/flows/user-settings)
+- [Account Verification](https://www.ory.sh/docs/kratos/self-service/flows/verify-email-account-activation)
```

Comment:
I don't think that both words in self-service flow names should be capitalized.

Reply:
Sure. Generally, it's good to use sentence case as much as possible.